templates/umap/navigation.html

```diff
@@ -0,0 +1,25 @@
+{% load static from staticfiles %}
```

Comment:
Not sure why we have navigation.html AND menu.html. Seems like this can be factorized in some way.